src/components/menu/Menu.tsx

```diff
@@ -8,18 +8,17 @@ import Widget from 'components/widget';
 
 import styles from './Menu.scss';
 import { hideMenu, setSelectionRange, showEditorAnnotation } from 'store/widgets/actions';
-import { Range } from 'xpath-range';
+import { Range as XPathRange } from 'xpath-range';
 import { PPScopeClass } from '../../class_consts';
 import ppGA from '../../pp-ga';
-import { SerializedRangeWithText } from '../../utils/annotations';
+import { AnnotationLocation } from '../../utils/annotations';
 
 interface IMenuProps {
   locationX: number;
   locationY: number;
-  range: Range.SerializedRange;
-  text: string;
+  annotationLocation: AnnotationLocation;
 
-  setSelectionRange: (range: SerializedRangeWithText) => void;
+  setSelectionRange: (range: AnnotationLocation) => void;
```

Comment:
How about going for it and updating `range` var to something? What do you think?

Reply:
Oh yes, surely. I just had to stop somewhere :)

Reply:
I'll improve on that very soon.